plenum/server/request_handlers/static_taa_helper.py

```diff
@@ -0,0 +1,70 @@
+from _sha256 import sha256
+from typing import Optional
+
+from common.serializers.serialization import config_state_serializer
+from plenum.common.constants import TXN_TYPE, TXN_AUTHOR_AGREEMENT, TXN_AUTHOR_AGREEMENT_AML, DOMAIN_LEDGER_ID
+from plenum.common.exceptions import UnauthorizedClientRequest
+from plenum.common.request import Request
+from plenum.server.request_handlers.utils import is_trustee
+
+MARKER_TAA = "2"
+MARKER_TAA_AML = "3"
```

Comment:
please, fix it the next PR.

Reply:
I would move every marker (for a txn type) into the corresponding WriteReqHandler.
Corresponding read req handlers (for this txn type) can get it from WriteReqHandlers (static method or constant).